hilbert_config/hilbert_cli_config.py

```diff
@@ -3422,7 +3422,7 @@ def yaml_dump(*args, **kwargs):
     kwargs['canonical'] = kwargs.pop('canonical', False)
 
     if pretty:
-        kwargs['version'] = kwargs.pop('version', (1,2))
+#        kwargs['version'] = kwargs.pop('version', (1,2))
```

Comment:
Why do you remove the version?

Reply:
YAML format is also used for pretty listing of stations, apps etc - version tag looks ugly there.
Also i did some experiments and did not notice any changes in the generated YAML for our files (save for that header)...

Reply:
So this is only for the `list` format? Then it's OK. (I didn't check the context, sorry.)

Reply:
Nope it is about general YAML output, which is currently also used for pretty printing of IDs by high level users commands of hilbert tool (e.g. `list_stations` etc.) by default. 
Note that those `list_*` commands have `--format` with choices: `list`, `yaml`. The yaml output is default for pretty output (and for consistency with previous user experience).